rewrites.js

```diff
@@ -167,9 +167,14 @@ exports.REWRITES = [
   // New Create Collective Flow
   {
     source:
-      '/:hostCollectiveSlug?/:verb(apply|create)/:version(v2)?/:category(opensource|community|climate|covid-19)?/:step(form)?',
+      '/:hostCollectiveSlug?/:verb(apply/new|create)/:version(v2)?/:category(opensource|community|climate|covid-19)?/:step(form)?',
```

Comment:
In the end, it's a bit weird to have `apply/new` as a parameter (having a slash in a parameter value doesn't look like a good practice). Can we use a different parameter. Put this in `:step` or introduce another parameter?

In the end maybe instead of `apply/new`, simply using `apply/form` is what we're looking for? I'm not sure, I might be missing something.

Reply:
using `apply/form` will directly lead to the `create-collective` form here, which should be the next step after clicking `request manual verification` here https://opencollective.com/opensource/apply/form 

Though moving `new` in `:step` will do the job.

Reply:
moving `new` in `:step` wasn't working as expected, so I added `new` seperately-

`:verb(apply|create)/new/:version(v2)?`

Reply:
Hmm, this can have side effects -

- `:verb(apply|create)` here will redirect `/apply` to `create-collective` instead of showing modal. 
-  Can't move `new` to `:step` or any other optional parameter as it will still redirect `/apply` to `/create-collective`
-  We can't introduce a new required parameter as `:verb(apply|create)/new/:version(v2)?` as we will have to update all URLs with `/create/new` `create/new/opensource` etc.


I think the safest option will be changing `/apply` to `/apply-new` to make `/apply` free - 

`:verb(apply-new|create)`

/cc @znarf

Reply:
Or we can add a separate rewrite for `/apply/new` -

```js
{
    source: '/:hostCollectiveSlug?/apply/new',
    destination: '/create-collective',
  },
```